adapters/adnuntius/adnuntiustest/supplemental/width-error.json

```diff
@@ -0,0 +1,84 @@
+{
+    "mockBidRequest": {
+        "id": "test-request-id",
+        "site": {
+            "page": "prebid.org"
+        },
+        "imp": [
+            {
+                "id": "test-imp-id",
+                "banner": {
+                    "format": [
+                        {
+                            "w": 300,
+                            "h": 250
+                        },
+                        {
+                            "w": 300,
+                            "h": 600
+                        }
+                    ]
+                },
+                "ext": {
+                    "bidder": {
+                        "auId": "123"
+                    }
+                }
+            }
+        ]
+    },
+
+    "httpCalls": [
+        {
+            "expectedRequest": {
+                "uri": "http://whatever.url?format=json&tzo=0",
+                "body": {
+                    "adUnits": [
+                        {
+                            "auId": "123",
+                            "targetId": "123-test-imp-id"
+                        }
+                    ],
+                    "context": "prebid.org",
+                    "metaData": {}
+                }
+            },
+            "mockResponse": {
+                "status": 200,
+                "body": {
+                    "adUnits": [
+                        {
+                            "auId": "0000000000000123",
+                            "targetId": "123-test-imp-id",
+                            "html": "<ADCODE>",
+                            "responseId": "adn-rsp-900646517",
+                            "ads": [
+                                {
+                                    "destinationUrls": {
+                                        "url": "http://www.google.com"
+                                    },
+                                    "bid": {
+                                        "amount": 20.0,
+                                        "currency": "NOK"
+                                    },
+                                    "adId": "adn-id-1559784094",
+                                    "creativeWidth": 980,
```

Comment:
This is a valid scenario but you're not getting test coverage of lines `if Werr != nil {` and `if len(adsErr) > 0 {`. If you change this to a string that doesn't parse to an int (e.g. "ABC"), the call to `ParseInt` will return the error as `Werr` giving you the code coverage.